### PR TITLE
Outposts: updating RN xref to postinstall content

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.15

Issue:
[OSDOCS-9565](https://issues.redhat.com//browse/OSDOCS-9565)

Link to docs preview:
https://72145--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-installing-aws-outposts

QE review:
Not required

Additional information:
Hold for #71263 pick into 4.15
Updates #72113

### CODE REVIEW

**Review by kalexand-rh — APPROVED** [review]
When the build passes, please merge this. No QE is required.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -119,7 +119,7 @@ For more information, see xref:../installing/installing_aws/installing-aws-local
 
 In {product-title} version 4.14, you could install a cluster on AWS with compute nodes running in AWS Outposts as a Technology Preview. In {product-title} 4.15, you can install a cluster on AWS into an existing VPC and provision compute nodes on AWS Outposts as a postinstallation configuration task.
 
-For more information, see xref:../installing/installing_aws/installing-aws-vpc.adoc[Installing a cluster on AWS into an existing VPC].
+For more information, see xref:../installing/installing_aws/installing-aws-vpc.adoc[Installing a cluster on AWS into an existing VPC] and xref:../post_installation_configuration/configuring-aws-outposts.adoc[Extending an AWS VPC cluster into an AWS Outpost].
```
Comment (kalexand-rh) [inline release_notes/ocp-4-15-release-notes.adoc:122]: Both of these xrefs need anchors.

Comment (jeana-redhat) [reply release_notes/ocp-4-15-release-notes.adoc:122]: ACK

**Pull request merged**